java/src/org/openqa/selenium/remote/HttpCommandExecutor.java

```diff
@@ -46,7 +46,7 @@ public class HttpCommandExecutor implements CommandExecutor, NeedsLocalLogs {
   private final URL remoteServer;
   public final HttpClient client;
   private final HttpClient.Factory httpClientFactory;
-  private final Map<String, CommandInfo> additionalCommands;
+  protected final Map<String, CommandInfo> additionalCommands;
```

Comment:
If the intent is to make this available to the downstream children, why not expose a getter that returns an `Immutable` map ?

Reply:
Sure, this sounds good! I will make the changes

Reply:
@mykola-mokhnach I have pushed the changes as per this, can you confirm if this works for appium?

Reply:
thanks, this should work